tests/tasks/verify_settings.yml

```diff
@@ -21,9 +21,11 @@
     - name: Check if the set password matches the existing password
       vars:
         __ipaddress: >-
-          "{{ __mssql_custom_ip_address.stdout | regex_search('[0-9].*') | quote }}"
+          "{{ __mssql_custom_ip_address.stdout
+          | regex_search('[0-9].*') | quote }}"
         __tcpport: >-
-          "{{ __mssql_custom_tcp_port.stdout | regex_search('[0-9].*') | quote }}"
+          "{{ __mssql_custom_tcp_port.stdout
+          | regex_search('[0-9].*') | quote }}"
```

Comment:
you should reduce some of this code duplication between tasks/main.yml and tests/tasks/verify_settings.yml

since you are using `>-` here you don't need the double quotes `"` around the `{{ ... }}`